pom.xml

```diff
@@ -72,7 +72,7 @@
 		<dependency>
 			<groupId>org.elasticsearch</groupId>
 			<artifactId>elasticsearch</artifactId>
-			<version>1.4.0</version>
+			<version>1.5.2</version>
```

Comment:
I had issues with 1.4.0, don't know why it was not working so I upgraded to latest GA released.